### PR TITLE
Add i18n to the wbsgetsuggestions api module

### PR DESCRIPTION
**Pull request by filbertkm**
Reworded some of the messages to be more clear.

[Bug: T122041](http://phabricator.wikimedia.org/T122041)

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+        ]
+    },
+    "propertysuggester-desc": "Suggests properties when adding Wikibase statements.",
+    "apihelp-wbsgetsuggestions-description": "API module to get property suggestions for Wikibase entities.",
+    "apihelp-wbsgetsuggestions-example-1": "Get suggestions for entity Q4",
+    "apihelp-wbsgetsuggestions-example-2": "Get suggestions for entity Q4 from rank 10 to 15",
+    "apihelp-wbsgetsuggestions-example-3": "Get suggestions for the property combination P21 and P31",
+    "apihelp-wbsgetsuggestions-example-4": "Get suggestions for the qualifier that are used with P21",
+    "apihelp-wbsgetsuggestions-example-5": "Get suggestions for the references that are used with P21",
+    "apihelp-wbsgetsuggestions-param-context": "Either item, reference or qualifier.",
```
Comment (thiemowmde) [inline i18n/en.json]: This line does not add any knowledge. But well, I can see this was not created in this patch.

File: src/PropertySuggester/GetSuggestions.php [hunk 2]
```diff
@@ -211,45 +211,20 @@ public function getAllowedParams() {
...
 
 	/**
-	 * @see ApiBase::getParamDescription()
+	 * @see ApiBase::getExamplesMessages()
 	 */
-	public function getParamDescription() {
+	public function getExamplesMessages() {
 		return array(
-			'entity' => 'Suggest attributes for given entity',
-			'properties' => 'Identifier for the site on which the corresponding page resides',
```
Comment (thiemowmde) [inline src/PropertySuggester/GetSuggestions.php]: Oh, was this a bug?

File: i18n/en.json [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
-	},
-	"propertysuggester-desc": "Suggests properties when adding Wikibase statements."
+    "@metadata": {
+        "authors": [
+            "Christian Dullweber",
+            "Katie Filbert"
+        ]
+    },
+    "propertysuggester-desc": "Suggests properties when adding Wikibase statements.",
+    "apihelp-wbsgetsuggestions-description": "API module to get property suggestions for Wikibase entities.",
```
Comment (thiemowmde) [inline i18n/en.json]: The original message contained "when editing". I think this should not be removed. What about "API module to get suggestions for additional properties when editing Wikibase entities"?

**Comment by thiemowmde** [comment]
Minor suggestions, +2 otherwise.

File: i18n/en.json (hunk 3 above)
Comment (filbertkm) [inline i18n/en.json]: the api module is out of context of editing (although that is the intended use). i can try to reword and expand the description to be more clear.

File: src/PropertySuggester/GetSuggestions.php (hunk 2 above)
Comment (filbertkm) [inline src/PropertySuggester/GetSuggestions.php]: i think so. the text makes no sense here.

File: i18n/en.json (hunk 3 above)
Comment (thiemowmde) [inline i18n/en.json]: Technically yes, but practically? Isn't "these are missing, consider adding these" the only use of these suggestions?

**Comment by thiemowmde** [comment]
Will merge for now to get this unstuck. Please try to improve the wording of these messages in a later patch.

**Pull request merged**